docs/data/xrootd-authorization.md

```diff
@@ -0,0 +1,169 @@
+Configuring XRootD Authorization
+================================
+
+There are several authorization options in XRootD available through the security plugins. 
+In this document, we will cover the security option supported in the OSG:
+
+-   [`xrootd-lcmaps`](#recommended-security-option-xrootd-lcmaps-authorization): Using the LCMAPS callout to use
+    X509-based authentication
+
+Note: On the data nodes, the files will actually be owned by unix user `xrootd` (or other daemon user), not as the user
+authenticated to, under most circumstances. 
+XRootD will verify the permissions and authorization based on the user that the security plugin authenticates you to,
+but, internally, the data node files will be owned by the `xrootd` user.
+
+#### Authorization file
+
+In order to add security to your cluster you will need to add "auth\_file" on the your data server node. 
+Create `/etc/xrootd/auth_file` :
+
+```file
+# This means that all the users have read access to the datasets 
+u * %RED%/data/xrootdfs%ENDCOLOR% lr
+
+# This means that all the users have full access to their private dirs 
+u = %RED%/data/xrootdfs/%ENDCOLOR%@=/ a
+
+# This means that this privileged user can do everything 
+# You need at least one user like that, in order to create the 
+# private dir for each user willing to store his data in the facility 
+u xrootd %RED%/data/xrootdfs%ENDCOLOR% a 
+```
+
+Here we assume that your storage path is `/data/xrootdfs` (same as in the
+previous example).
+
+Change file ownership (if you have created file as root):
+
+```console
+root@host # chown xrootd:xrootd /etc/xrootd/auth_file
+```
+
+
+This file is a flat file of the following form:
+
+``` file
+idtype id path privs
+```
+
+Some examples of each option. For more details or examples on how to use
+templated user options, see [XRootd Authorization Database
+File](http://xrootd.org/doc/dev47/sec_config.htm#_Toc489606599).
+
```

Comment:
I don't see any examples here. Some that I'd suggest:
```
# any user can read files everywhere _except_ under /private
u * -rl /private / rl
# users can do anything to files and dirs under their home directories
u = /home/@=/ a
# VOMS group uscms can do anything to files and dirs under /uscms
g uscms /uscms a
```

Reply:
also you might want to put somewhere that specific paths need to be specified _before_ generic paths, i.e. `u * rl / -rl /private` does _not_ work.

Reply:
Thanks @matyasselmeci, i've pushed an update based on your feedback.

Reply:
This has been addressed in 0744f8c